catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/validation/impl/validator/EnumerationValidator.java

```diff
@@ -37,6 +37,8 @@
  * Validates an attribute's value(s) against a set of acceptable values.
  */
 public class EnumerationValidator implements AttributeValidator {
+    private final boolean ignoreCase;
```

Comment:
`ignoreCase` is used in `validate()` to determine whether the attribute value should be set to all lower case before checking it against the set of values.